crates/aptos/src/node/local_testnet/node.rs

```diff
@@ -155,7 +155,12 @@ impl NodeManager {
 
         // So long as the indexer relies on storage indexing tables, this must be set
         // for the indexer GRPC stream on the node to work.
-        node_config.storage.enable_indexer = run_txn_stream;
+        // node_config.storage.enable_indexer = run_txn_stream;
+        node_config.indexer_table_info.table_info_service_mode = match run_txn_stream {
+            // Local not should be be responsible for backup or restore of table info tables.
```

Comment:
Sentence is confusing
```suggestion
            // Localnet should be responsible for backup or restore of table info tables.
```

Reply:
Lightspeed!! thanks for catching it... was testing haha 😆 

Going to move back to draft